src/main/java/other/BitcoinService.java

```diff
@@ -17,28 +18,30 @@
     private static final RequestSpecification SPECIFICATION = new RequestSpecBuilder().setBaseUri(baseUri).build();
     private ObjectMapper objectMapper = new ObjectMapper();
 
-    public void getListOfAverages() {
-
+    public ArrayList getListOfAverages() {
 
         List<BitcoinDay> bitcoinDay = null;
+        ArrayList average = new ArrayList();
 
             bitcoinDay = readValue(HttpRequests.executeGet(SPECIFICATION,"indices/global/history/BTCUSD?period=alltime", 200), BitcoinDay.class);
 
         for (int i = 0; i < 30; i++) {
```

Comment:
why do we hardcode 30 here? it looks like this will not work for all months

Reply:
Not sure i understand your comment. Both services returns values for last 30 days, not for current month. Anyway, I've moved number of days to a variable